applications/FluidDynamicsApplication/tests/cpp_tests/test_symbolic_stokes_element.cpp

```diff
@@ -0,0 +1,328 @@
+            model_part.CreateNewNode(5, 1.0, 0.0, 1.0);
+            model_part.CreateNewNode(6, 0.0, 1.0, 1.0);
+            model_part.CreateNewElement("SymbolicStokes3D6N", 1, {1, 2, 3, 4, 5, 6}, p_elem_prop);
+
+            // Set the nodal DENSITY and DYNAMICS_VISCOSITY
+            Element::Pointer p_element = model_part.pGetElement(1);
+            for (NodeIteratorType it_node = model_part.NodesBegin(); it_node != model_part.NodesEnd(); ++it_node)
+            {
+                it_node->FastGetSolutionStepValue(DENSITY) = p_elem_prop->GetValue(DENSITY);
+                it_node->FastGetSolutionStepValue(DYNAMIC_VISCOSITY) = p_elem_prop->GetValue(DYNAMIC_VISCOSITY);
+                if (it_node->Z() == 0.0)
+                {
+                    it_node->Fix(VELOCITY_X);
+                    it_node->Fix(VELOCITY_Y);
+                    it_node->Fix(VELOCITY_Z);
+                }
+                if (it_node->Z() == 1.0)
+                {
+                    it_node->FastGetSolutionStepValue(VELOCITY_X) = 0.1;
+                    it_node->FastGetSolutionStepValue(VELOCITY_Y) = 0.1;
+                    it_node->FastGetSolutionStepValue(VELOCITY_Z) = 0.1;
+                }
+            }
+
+            // Compute RHS and LHS
+            Vector RHS = ZeroVector(24);
+            Matrix LHS = ZeroMatrix(24, 24);
+            p_element->Initialize();
+            p_element->CalculateLocalSystem(LHS, RHS, process_info);
+
+            // Check the RHS values
+            KRATOS_CHECK_NEAR(RHS(0), -16.66666667, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(1), -16.66666667, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(2), 47.22222222, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(3), 0.03854166667, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(4), -33.33333333, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(5), -25, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(6), -27.77777778, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(7), -0.008333333333, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(8), -25, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(9), -33.33333333, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(10), -27.77777778, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(11), -0.008333333333, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(12), -91.66666667, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(13), -91.66666667, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(14), -72.22222222, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(15), 0.03854166667, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(16), -108.3333333, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(17), -100, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(18), -147.2222222, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(19), -0.05520833333, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(20), -100, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(21), -108.3333333, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(22), -147.2222222, 1e-07);
+            KRATOS_CHECK_NEAR(RHS(23), -0.05520833333, 1e-07);

```

Comment:
please use the check macros for [vectors and matrices](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/includes/checks.h#L53-L151)
Macro unrolling is expensive and increases compilation time

Reply:
done